index.html

```diff
@@ -2,316 +2,349 @@
+            <img class="snapshoot-style on-large" src="images/Snapshoot Portfolio5.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style">
+                    Tonic
+                </h1>
+                <section class="card-description">
+                    <ul class="font-style">
+                        <li class="card-list-title"> CANOPY</li>
+                        <li class="Job-title_and_year">
+                            Back End Dev
+                        </li>
+                        <li class="Job-title_and_year">
+                            2022
+                        </li>
+                    </ul>
+                </section>
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button button" type="button">See Project</button>
+            </section>
+        </section>
+        <section class="card fourth-card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio4.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style on-small">
+                    Multi-Post <br> Stories
+                </h1>
+                <h1 class="card-header font-style on-large">
+                    Multi-Post Stories
+                </h1>
+                <section class="card-description">
 
-	</div>
+                    <ul class="font-style">
+                        <li class="card-list-title on-small"> CANOPY</li>
+                        <li class="on-large card-list-title ">FACEBOOK</li>
+                        <li class="on-large Job-title_and_year">Full Stack Dev</li>
+                        <li class="Job-title_and_year on-small">
+                            Back End Dev
+                        </li>
+                        <li class="Job-title_and_year">
+                            2022
+                        </li>
+                    </ul>
+                </section>
+                <p class="notice font-style">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programming-languages font-style">
+                    <li>html</li>
+                    <li class="on-large">Ruby On Rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project-button button" type="button">See Project</button>
+            </section>
+        </section>
+    </div>

```

Comment:
- [ ] I would kindly suggest you not to use a sections inside a section tags its not semantic to wrap a `section tag` inside a `section tag`kindly re organise your code in a semantic way so that it will be easy to read for other developers.